JPetTaskIO/JPetTaskIO.cpp

```diff
@@ -207,10 +211,12 @@ bool JPetTaskIO::createOutputObjects(const char* outputFilename)
 {
   fWriter = new JPetWriter( outputFilename );
   assert(fWriter);
-  if (fSubTask) {
+  if (!fSubTasks.empty()) {
     //auto task = std::dynamic_pointer_cast<JPetTask>(fTask);
-    auto task = dynamic_cast<JPetUserTask*>(fSubTask.get());
-    task->setStatistics(std::move(fStatistics));
+    for (auto fSubTask : fSubTasks) {
+      auto task = dynamic_cast<JPetUserTask*>(fSubTask.get());
+      task->setStatistics(std::move(fStatistics));
```

Comment:
This is probably a bug, since if you have more than one task, then the first one would take fStatistics via std::move and the second one would get the nullptr.